packages/f-content-cards/src/components/cardTemplates/SkeletonLoader.vue

```diff
@@ -17,6 +17,10 @@ export default {
         count: {
             type: Number,
             default: 1
+        },
+        type: {
+            type: String,
```

Comment:
Might be worth adding a [validation](https://vuejs.org/v2/guide/components-props.html#Prop-Validation) function here, to ensure we're allowing recognised types only:

```suggestion
            type: String,
            validator (value) {
                return ['promo', 'postOrder'].indexOf(value) !== -1
            },
```